### ISSUE TITLE
salt provisioner ignoring output settings

### ISSUE THREAD

**Issue by hexedpackets** [mitchellh/vagrant#6153]
I have salt setup in my Vagrantfile like this:

```ruby
config.vm.define "generic" do |node|
  node.vm.host_name = "test.generic.0"

  node.vm.provision :salt do |salt|
    salt.masterless = true
    salt.run_highstate = true
    salt.colorize = true
    salt.log_level = "info"
    salt.verbose = true
  end
end
```

Running `vagrant up` or `vagrant provision` ignores both the log_level and colorize options. Verbose does work, but its very hard to parse through as my terminal gets flooded with debug messages.

**Comment by hexedpackets** [comment]
I figured out the colorize option. Salt requires having a TTY for colorized output, so I needed to set `config.ssh.pty = true`. However, the log_level is still running running at debug.

**Comment by msiebeneicher** [comment]
+1

**Comment by sethvargo** [comment]
This should be fixed in 1.8.3. Thanks!
